client/js/bar.js

```diff
@@ -4,13 +4,27 @@ const svg = document.createElementNS('http://www.w3.org/2000/svg', 'svg');
 svg.setAttribute('viewBox', '0 0 100 100'); // change height in css
 document.getElementById('graph').appendChild(svg);
 
+// textBlock should align vertically with bar chart
+const barTextContainer = document.getElementById('bar-text');
+barTextContainer.setAttribute('height', svg.getBoundingClientRect().height);
```

Comment:
This is a bit fragile. You should be able to solve this with css. Try something like `height: 100%` and let me know if you need help debugging.

Reply:
TODO on tuesday